packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -340,14 +342,11 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(props, ref) {
       document.body.addEventListener('touchstart', handleBodyTouchStart);
       document.body.addEventListener('touchmove', handleBodyTouchMove, { passive: false });
       document.body.addEventListener('touchend', handleBodyTouchEnd);
-      // https://plus.google.com/+PaulIrish/posts/KTwfn1Y2238
```

Comment:
Google+ is out, an older version can be found at https://web.archive.org/web/20150608114000/https://plus.google.com/+PaulIrish/posts/KTwfn1Y2238. I believe we can ignore the problem. I haven't seen any recent resource about it. Watch https://www.youtube.com/watch?v=F3A6Skckh9c, no mention of the problem.